src/features/asciidoctorConfig.ts

```diff
@@ -0,0 +1,25 @@
+import * as vscode from 'vscode'
```

Comment:
Should we add the license header?

Reply:
To be honest, I want to remove the license header everywhere. In my opinion, it adds unnecessary noise.
The extension is based on https://github.com/microsoft/vscode/tree/main/extensions/markdown-language-features but the license is MIT so https://github.com/asciidoctor/asciidoctor-vscode/blob/master/LICENSE should be enough.